### ISSUE TITLE
What is the base learning rate for batch size 512?

### ISSUE THREAD

**Issue by cneyang** [facebookresearch/vicreg#23]
The original paper states that the base lr is 0.4.
![image](https://user-images.githubusercontent.com/50402681/226258071-563f88eb-ad39-4ab8-9b83-6e06e3056c04.png)

However, 8-GPU single node training script says 0.3.
```
python -m torch.distributed.launch --nproc_per_node=8 main_vicreg.py --data-dir /path/to/imagenet/ --exp-dir /path/to/experiment/ --arch resnet50 --epochs 100 --batch-size 512 --base-lr 0.3
```

Which one is correct?
